data/ka'het missions.txt

```diff
@@ -110,12 +111,14 @@ mission "First Contact: Ka'het: Quarg 1"
 			`	Apart from the utter silence, which is pretty common in most rings, you are convinced by the fact that every room, apartment and cabin has been carefully emptied, leaving nothing behind. Perhaps it would be wise to ask the Quarg what happened here...`
 				accept
 
-mission "First Contact: Ka'het: Quarg 2"
+# The Z ensures this happens after meeting the Quarg.
```

Comment:
Given that mission order by name is now mentioned in the wiki, I don't think this comment is necessary.

Reply:
fixed